packages/kbn-search-index-documents/lib/fetch_search_results.ts

```diff
@@ -27,6 +28,7 @@ export const fetchSearchResults = async (
         index: indexName,
         size,
         ...(!!query ? { q: escapeLuceneChars(query) } : {}),
+        track_total_hits: trackTotalHits,
```

Comment:
why are we using this vs fetching `GET /<index>/_count` ? 

Isn't this an inaccurate count of just the total hits of the search?

Reply:
So with `track_total_hits: true`, it will produce an accurate match of hits https://www.elastic.co/guide/en/elasticsearch/reference/8.15/search-your-data.html#track-total-hits 

I believe behind the scenes, _count uses this path as you specify a query with it. 

Other options I considered:
- i didn't want to impact the current index API (index_management/indices) to get _count as its also used in the list API (and involve performing a count query for each index) and unlikely to be approved
- I didn't want to make an extra REST call to our own serverless_search APIs which rely on _count and may lead to inconsistency 

If we do incorporate search functionality into docs then yes this will need to be changed. For now I think this makes sense vs eventually building our own index endpoint for this.